config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,148 @@
+SimpleForm.setup do |config|
```

Comment:
I still know very little about 508 compliance, so just wondering if there might be problems with using bootstrap in that context? I read somewhere that bootstrap isn't the greatest for accessibility, but I am not sure what using `simple_form_bootstrap` is doing exactly here so not sure if that will be a problem